tests/unit/test_contracts/parameter_tests.cpp

```diff
@@ -12,9 +12,9 @@ namespace eosio {
      __attribute__((eosio_wasm_import))
      uint32_t get_blockchain_parameters_packed(char*, uint32_t);
      __attribute__((eosio_wasm_import))
-     void set_parameters_packed(char*, uint32_t);
+     void set_parameters_packed(const char*, uint32_t);
```

Comment:
`set_parameters_packed` and (below) `get_parameters_packed` are not used in this test, are they needed here? ((I know this is not the problem this PR trying to fix, just want to check)

Reply:
The functions to be test are set/get_parameters, they will call set/get_parameters_packed, so they are needed be there.